manuscript/090_How_to_start.md

```diff
@@ -365,7 +369,7 @@ public void TODO()
 }
 ```
 
-Now, this compiles but is evaluated as false because of the guard assertion that we put at the end. Our goal is to substitute it with a�real assertion for a�real expected result. The return value of the `HoldOnASecondWhileWeImportYourDatabase` is a�string message, so all we need to do is to come up with the message that we expect in case of trial version:
+Now, this compiles but is evaluated as false because of the guard assertion that we put at the end. Our goal is to substitute it with a proper assertion for the expected result. The return value of the call to `HoldOnASecondWhileWeImportYourDatabase` is a string message, so all we need to do is to come up with the message that we expect in case of the trial version:
```

Comment:
By the way, I just noticed my mistake. I wrote here that the Statement is false because of the guard assertion, but the snippet for the initial implementation shows that there's a NotImplementedException thrown even earlier. Can you change the following:

```csharp
public class TrialVersionMessages : Messages
{
 public string HoldOnASecondWhileWeImportYourDatabase(string userName)
 {
   throw new NotImplementedException();
 }
}
```
to
```csharp
public class TrialVersionMessages : Messages
{
 public string HoldOnASecondWhileWeImportYourDatabase(string userName)
 {
   return null;
 }
}
```